lib/performance_data/metrics.rb

```diff
@@ -16,6 +16,18 @@ def problem_reports_weekly_average
       average(@data[:problem_reports]) * 7
     end
 
+    def unique_pageviews_multipart_average
```

Comment:
I think there's another class trying to get out of this one (something like `MultiPartMetrics` maybe?). It could have the same method signature, and that'd save on a lot of ugly conditional logic in the views.